server/utils/tasks/seed-cache.ts

```diff
@@ -12,6 +12,10 @@ export async function seedCache() {
     .from(metaDataTable)
     .where(inArray(metaDataTable.key, ['seoConfig', 'careerSiteConfig', 'firstSetupAccessKey']));
 
+  //getting the review tags from the reviewTagsTable
+  const reviewTags = await db.select().from(reviewTagsTable);
+  console.log(reviewTags); //debug to check the reviewtags response
```

Comment:
delete this

Reply:
@amandesai01 you mean the comments or code?

Reply:
remove the comments and logging